### PR TITLE
[Backend] - Multitable Comment Resolution

### PR DESCRIPTION
**Pull request by drewjhart**

### CODE REVIEW

File: networking/amplify/backend/api/mobile/schema.graphql [hunk 1]
```diff
@@ -30,8 +30,6 @@ type GameTemplate @model{
   phaseTwoTime: Int!
```
Comment (drewjhart) [inline networking/amplify/backend/api/mobile/schema.graphql:30]: nonnull

File: networking/amplify/backend/api/mobile/schema.graphql [hunk 2]
```diff
@@ -30,8 +30,6 @@ type GameTemplate @model{
...
   imageUrl: String
   questionTemplates: [QuestionTemplate] @manyToMany(relationName: "GameQuestions")
-  createdAt: AWSDateTime
-  updatedAt: AWSDateTime
 }
```
Comment (drewjhart) [inline networking/amplify/backend/api/mobile/schema.graphql:33]: createdAt/updatedAt removed

File: networking/amplify/backend/api/mobile/schema.graphql [hunk 3]
```diff
@@ -55,8 +53,6 @@ type QuestionTemplate @model {
   standard: String
   imageUrl: String
   gameTemplates: [GameTemplate] @manyToMany(relationName: "GameQuestions")
-  createdAt: AWSDateTime
-  updatedAt: AWSDateTime
 }
```
Comment (drewjhart) [inline networking/amplify/backend/api/mobile/schema.graphql:56]: createdAt/updatedAt removed

File: networking/src/ApiClient.ts [hunk 4]
```diff
@@ -677,7 +676,9 @@ class GameTemplateParser {
                     const { gameTemplates, ...rest } = item.questionTemplate;
                     // Only add to questionTemplates if 'rest' is not empty
                     if (Object.keys(rest).length > 0) {
-                        questionTemplates.push({questionTemplate: rest as IQuestionTemplate, gameQuestionId: item.id as string});
```
Comment (drewjhart) [inline networking/src/ApiClient.ts:680]: parsing for createdAt/updatedAt string -> Date starts here

File: networking/src/Parsers/QuestionTemplateParser.ts [hunk 5]
```diff
@@ -41,7 +41,13 @@ export class QuestionTemplateParser {
       if (isNullOrUndefined(id) ||
           isNullOrUndefined(title) ||
           isNullOrUndefined(owner) ||
-          isNullOrUndefined(version)) {
+          isNullOrUndefined(version) ||
+          isNullOrUndefined(domain) ||
+          isNullOrUndefined(cluster) ||
+          isNullOrUndefined(grade) ||
+          isNullOrUndefined(standard) ||
```
Comment (maniramezan) [inline networking/src/Parsers/QuestionTemplateParser.ts:48]: Are these all reqiured? I remember we discussed some CCSS fields might not be available for certain questions

**Pull request merged**